pom.xml

```diff
@@ -0,0 +1,105 @@
+
+    <name>securebanking-openbanking-uk-rs</name>
+    <description>A UK Open Banking Resource Server for the Secure Banking Accelerator Toolkit</description>
+
+    <modules>
+        <module>securebanking-openbanking-uk-rs-admin-api</module>
+        <module>securebanking-openbanking-uk-rs-backoffice-api</module>
+        <module>securebanking-openbanking-uk-rs-obie-api</module>
+        <module>securebanking-openbanking-uk-rs-simulator</module>
+        <module>securebanking-openbanking-uk-rs-simulator-app</module>
+    </modules>
+
+    <parent>
+        <groupId>com.forgerock.securebanking</groupId>
+        <artifactId>securebanking-parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+    </parent>
+
+    <dependencyManagement>
+        <dependencies>
+            <!-- ForgeRock BOM -->
+            <dependency>
+                <groupId>com.forgerock.securebanking</groupId>
+                <artifactId>securebanking-openbanking-uk-bom</artifactId>
+                <version>1.0.0-SNAPSHOT</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <scm>
+        <connection>scm:git:git@github.com:SecureBankingAcceleratorToolkit/securebanking-openbanking-uk-rs.git
+        </connection>
+        <developerConnection>scm:git:git@github.com:SecureBankingAcceleratorToolkit/securebanking-openbanking-uk-rs.git
+        </developerConnection>
+        <url>https://github.com/SecureBankingAcceleratorToolkit/securebanking-openbanking-uk-rs.git</url>
+        <tag>HEAD</tag>
+    </scm>
+
+    <distributionManagement>
+        <repository>
+            <id>maven.forgerock.org-community</id>
+            <name>maven.forgerock.org-releases</name>
+            <url>https://maven.forgerock.org:443/repo/community</url>
+        </repository>
+        <snapshotRepository>
+            <id>maven.forgerock.org-community-snapshots</id>
+            <name>maven.forgerock.org-snapshots</name>
+            <url>https://maven.forgerock.org:443/repo/community</url>
+        </snapshotRepository>
+    </distributionManagement>
+
+    <repositories>
+        <repository>
+            <id>maven.forgerock.org-community</id>
+            <name>maven.forgerock.org-releases</name>
+            <url>https://maven.forgerock.org:443/repo/community</url>
+        </repository>
+        <repository>
+            <id>maven.forgerock.org-community-snapshots</id>
+            <name>maven.forgerock.org-snapshots</name>
+            <url>https://maven.forgerock.org:443/repo/community</url>
+        </repository>
+        <repository>
+            <id>central</id>
+            <name>Central Repository</name>
+            <url>https://repo1.maven.org/maven2</url>
+            <layout>default</layout>
+            <snapshots>
+                <enabled>false</enabled>
+            </snapshots>
+        </repository>
+        <repository>
```

Comment:
I think that we can delete the bintray repository.

Reply:
Done